tasks/post_to_catalog/post_to_catalog.py

```diff
@@ -189,7 +189,8 @@ def create_collection_sql():
     """
     )
 
-def create_granule_sql():
+def create_granule_sql() -> text:  # pragma: no cover
+    return 5
```

Comment:
typo?

Reply:
Whoops. Not sure how I managed that one. Removed.